modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -35,21 +35,26 @@ This {release} version of the software and programming language should not be us
 
 The {release} release includes the following new features and capabilities:
 
-- Web interface for testing canisters.
+- The Candid web interface now has a "Lucky" button that can generate random values when the input field is empty.
+- Various updates to JS userlib, including:
+* Unit is renamed to Null
+* Support Principal type
+* Support service and function references
+- Motoko now supports type argument inference for automatic instantiation of type-parametric functions.
+- Native support for Principal type in Candid (IDL).
```

Comment:
I think we don't have any documentation for principal and reference types in the language manual.

Reply:
Actually Principal types are in the manual now. Will check on reference types.

Reply:
The manual states that shared function types and actor types are sharable, so we are good, I think, unless you meant something else.

Reply:
Cool! Maybe we can put a link in the release notes. People probably don't know what we are talking about otherwise.

Reply:
I'll add a line to my suggestion below. Not sure this is actually new to this release rather than just documented ;->

Reply:
Done